app/src/androidTest/java/org/dhis2/usescases/datasets/DataSetTest.kt

```diff
@@ -59,15 +69,68 @@ class DataSetTest : BaseTest() {
 
         // Step - Test combination of filters - TODO Move the step after creating dataset instance
         // ORG unit add some dataset instance out of Ngelahun CHC to filter by Ngelahun CHC
-        // Period filter from - to specific period where instansces exist
+        // Period filter from - to specific period where instances exist
         // Sync move after create dataset instance and check the filter afterwards
         // checkFilterCombination(orgUnit)
     }
 
-    private suspend fun enterDataSetStep() {
+    @Test
+    fun formConfigurationTestAutomate() = runTest {
+        enableFeatureConfigValue(Feature.COMPOSE_AGGREGATES_SCREEN)
+        // Start Activity
+        enterDataSetStep("DMicXfEri6s", "Form configuration options")
+
+
+        // Step - ANDROAPP-6795 Check content boxes above and below the table
+        checkContentBoxesAreDisplayed()
+        // Step - ANDROAPP-6810 Move a category to rows (click on sections 8, 16, 24)
+        // Step - ANDROAPP-6828 Automatic grouping (click on sections 19, 20, 22)
+        // Step - ANDROAPP-6811 Pivot options (click on sections 5, 13, 23)
+    }
+
+
+    @OptIn(ExperimentalTestApi::class)
+    private suspend fun checkContentBoxesAreDisplayed() {
+        composeTestRule.awaitIdle()
+        dataSetRobot {
+            clickOnDataSetAtPosition(0)
+        }
+        tableIsVisible()
+        // Check top and bottom content is displayed in initial section
+        dataSetDetailRobot(composeTestRule) {
+            composeTestRule.onNodeWithText("CONTENT BEFORE 1:", true)
```

Comment:
You are not using the robots. All this uses of composeTest rule should be inside the appropiate robot.